frontend/src/components/Meta.tsx

```diff
@@ -0,0 +1,34 @@
+import Head from 'next/head';
+
+interface Props {
+	title?: string;
+	keywords?: string;
+	description?: string;
+}
+
+const Meta: React.FunctionComponent<Props> = ({
+	title,
+	keywords,
+	description,
+}: Props) => {
+	return (
+		<Head>
+			<meta name='viewport' content='width=device-width, initial-scale=1.0' />
+			<meta name='keywords' content={keywords} />
+			<meta name='description' content={description} />
+			<meta charSet='utf-8' />
+			<link rel='icon' href='/favicon.ico' />
+			<title>{title}</title>
+		</Head>
+	);
+};
+
+Meta.defaultProps = {
```

Comment:
Getting an interesting note on `defaultProps`:
```
Warning: Meta: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.
    at Meta (webpack-internal:///./src/components/Meta.tsx:11:17)
    at Layout (webpack-internal:///./src/components/Layout/Layout.tsx:20:19)
    at App (webpack-internal:///./src/pages/_app.tsx:13:16)
    at StyleRegistry (/Users/jumar/src/elonsoc/ods/frontend/node_modules/.pnpm/styled-jsx@5.1.1_react@18.2.0/node_modules/styled-jsx/dist/index/index.js:449:36)
    at PathnameContextProviderAdapter (/Users/jumar/src/elonsoc/ods/frontend/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/shared/lib/router/adapters.js:60:11)
    at AppContainer (/Users/jumar/src/elonsoc/ods/frontend/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/server/render.js:291:29)
    at AppContainerWithIsomorphicFiberStructure (/Users/jumar/src/elonsoc/ods/frontend/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/server/render.js:327:57)
    at div
    at Body (/Users/jumar/src/elonsoc/ods/frontend/node_modules/.pnpm/next@13.2.1_biqbaboplfbrettd7655fr4n2y/node_modules/next/dist/server/render.js:614:21)
```

This is great to have for SEO purposes but could we find a better way to do this?

Reply:
Great, will change to default params instead. Had previously thought defaultProps was the standard.